src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -268,7 +268,10 @@ void style_transfer::init_options(
   options.create_integer_option(
       "num_styles", "The number of styles present in the model", FLEX_UNDEFINED,
       1, std::numeric_limits<int>::max());
-
+  options.create_boolean_option(
+      "verbose", "When set to true, verbose is printed", true, true);
+  options.create_string_option("content_feature", "Name of the content column", "image", true);
+  options.create_string_option("style_feature", "Name of the style column", "image", true);
```

Comment:
Why are these needed? I thought we were passing the data as two SArrays now, not as SFrames with column names.

What happens if both of these are left as the default? Isn't that a namespace collision of sorts?

Reply:
This is for the python side where the api still takes SFrames. Should we modify the api in Python as well?

Reply:
Python frontend is not changed so that still accepts SFrames for content and style. I am passing the relevant column as SArray to C++ but then since user passed content/style as SFrame, they would want to know the column names after loading the model.